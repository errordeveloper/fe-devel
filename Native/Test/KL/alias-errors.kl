/*
 *  Copyright 2010-2012 Fabric Engine Inc. All rights reserved.
 */

alias UnregisteredType AliasTypeOne;
alias Integer AliasTypeTwo;
alias Scalar AliasTypeTwo;
