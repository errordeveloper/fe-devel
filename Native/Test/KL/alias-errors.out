alias-errors.kl:5:1: error: 'UnregisteredType': type not registered
alias-errors.kl:7:1: error: type 'AliasTypeTwo' already registered with a different definition
